schemaless/create_uuid_map.py

```diff
@@ -27,8 +27,10 @@ def from_files(cls, schemaless_file, uuid_map_file):
 
         # Create a mapping between permit numbers and their associated PPTS
         # record (so we an ensure they are assigned the same UUID).
+        prj_to_ppts = {}
         permit_number_to_ppts = defaultdict(list)
         for fk, record in latest_records.get(PPTS.NAME, {}).items():
+            prj_to_ppts[record['record_id']] = fk
```

Comment:
nit on the naming: prj_to_ppts is a bit misleading since there are other record types that might be keyed right? like ppas/cuas? maybe ppts_id_to_fk or something is clearer?

Reply:
This dict should always be to ppts id, though, since it's only reading through the PPTS source